cms/templates/js/course-instructor-details.underscore

```diff
@@ -0,0 +1,38 @@
+  <div class="field text field-course-instructor-name">
+  	<label for="course-instructor-name-<%- index %>"><%= gettext("Name") %></label>
```

Comment:
Note that strings are not properly escaped in this file.

Also, if you nest inputs within labels you can get rid of the need for the linking by IDs (we try to avoid IDs).